python/python_fastapi_server/routers/edr.py

```diff
@@ -420,11 +471,16 @@ def get_params_for_collection(edr_collection: str) -> dict[str, Parameter]:
         else:
             label = param_el["name"]
 
+        symbol = Symbol(value=param_el["unit"], type=SYMBOL_TYPE_URL)
+        param_id = param_el["name"]
+        # If standard name was configured, use that instead with a vocabulary
+        if "standard_name" in param_el:
+            param_id = VOCAB_ENDPOINT_URL + param_el["standard_name"]
         param = Parameter(
             id=param_el["name"],
```

Comment:
So we are not yet adding a `description`, like in the example in #359.

Reply:
No indeed. This could be the title of the WMS Layer I presume?